spec/lib/salus/scanners/language_version/go_version_scanner_spec.rb

```diff
@@ -0,0 +1,87 @@
+require_relative '../../../../spec_helper.rb'
+
+describe Salus::Scanners::LanguageVersion::GoVersionScanner do
+  describe '#run' do
+    context 'with non-go project' do
+      it 'should record error to scanner report' do
+        repo = Salus::Repo.new('spec/fixtures/blank_repository')
+        scanner = described_class.new(repository: repo, config: {})
+
+        expect(scanner.should_run?).to eq(false)
+
+        scanner.run
+
+        expect(scanner.report.passed?).to eq(false)
+
+        msg = scanner.report.to_h.fetch(:errors)[0][:message]
+        expect(msg).to include('Please supply the path to a go application')
+      end
+    end
+
+    context 'with go project' do
+      let(:path_str) { "../../../../../spec/fixtures/language_version/go_version_scanner" }
+      let(:fixture_path) { File.expand_path(path_str, __dir__) }
+
+      it 'should check the version successfully' do
```

Comment:
This is validating that 
`go 1.16
`
passes for the following config:

```
scanner_configs:
  GoVersionScanner:
    min_version: '1.15.0'
    max_version: '1.20.3'
```
Two asks...
1) Could we add one more check where the version == either the min or max version?
For ex:
```scanner_configs:
  GoVersionScanner:
    min_version: '1.16.0'
    max_version: '1.20.3'
```
Note the 1.16 v 1.16.0 we are testing.

2.  Could you just add a quick one line comment on these tests for the version vs config version being tested? That'll save hopping around to the fixtures to grep each test
# go 1.16 vs config of min_version: '1.15.0' max_version: '1.20.3'

Reply:
Updated this part for both `go` and `ruby`.